test/models/user_test.rb

```diff
@@ -83,6 +83,26 @@ class UserTest < ActiveSupport::TestCase
     assert_equal [report: nil, date: Date.today, emotion: nil], user.reports_date_and_emotion(0)
   end
 
+  test "#depressed?" do
+    user = users(:hatsuno)
+    3.times do |i|
+      report = Report.new(
+        user_id: user.id, title: "test #{i}", description: "test",
+        wip: false, emotion: "sad", reported_on: Date.today - i
```

Comment:
```suggestion
        wip: false, emotion: "sad", reported_on: Date.current - i
```

Date.currentだとTimeZoneを考慮してくれます。RailsではDate.currentやTime.currentを使うと便利です。

Reply:
ありがとうございます。修正しました！